modules/cli-manager-custom-index-krew.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-custom-index_{context}"]
+= Adding the CLI Manager custom index to Krew
+
+Use the terminal to add the CLI manager custom index to Krew.
```

Comment:
We talked about this too, but should add some details on why you want (need?) to do this. Is this optional, or is this a required step?